### DISCUSSION TITLE
Orgs admin api not working

### DISCUSSION THREAD

**Discussion by satishviswanathan** [zitadel/zitadel#7701] [Q&A]
I have a UI workflow in which user get's authenticated and then I'm trying to use the same token to call the admin end point to get the list of organization that the user have access too.  I'm always getting the response as "No matching permissions found". 

**_Authorize End point_**

`https://<ZITADEL_DOMAIN>/oauth/v2/authorize?client_id=<CLIENT_ID>&redirect_uri=https%3A%2F%2Foidcdebugger.com%2Fdebug&scope=openid%20profile%20urn%3Azitadel%3Aiam%3Aorg%3Aproject%3Aid%3Azitadel%3Aaud&response_type=code&response_mode=query&code_challenge_method=S256&code_challenge=<CODE_CHALLENGE>`

**_Get Orgs end point_**

```
curl --location --request GET '<ZITADEL_DOMAIN>/management/v1/orgs/me' \
--header 'Authorization: Bearer <Token>' \
--data-urlencode 'client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer' \
--data-urlencode 'token=<TOKEN>'
```

Is this achievable , if so not sure what I'm missing here.  @dakshitha can you help me on this.

**Comment by dakshitha** [comment]
So the service user or the human user has to have the right level of permissions to call the ZITADEL management API (e.g., <ZITADEL_DOMAIN>/management/v1/orgs/me and others). Make sure you set the permissions, ORG_OWNER for example. 

See here for more details - https://zitadel.com/docs/guides/manage/console/managers

**Reply by satishviswanathan** [reply]
Thank you for sharing the info. Is there a way I can get list of orgs that I'm part of?

can I see the list of orgs flattened in the response of this end point ? @dakshitha do you think it's possible ?

https://<ZITADEL_DOMAIN>/oidc/v1/userinfo

**Reply by arun-pirai** [reply]
I'm also looking for a way to retrieve associated org for a logged in user. Could anyone pls suggest a way?
